server/backend/quiz/views.py

```
@@ -56,6 +56,16 @@ def post(self, request):
             return Response(serializer.data, status=status.HTTP_201_CREATED)
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+    
+from typing import ClassVar
```

Ruff (0.8.2)

Module level import not at top of file

(E402)